variables.tf

```
@@ -0,0 +1,36 @@
+variable "resource_group_name" {
+  description = "Назва ІСНУЮЧОЇ групи ресурсів"
+  type        = string
+}
+
+variable "virtual_network_name" {
+  description = "Назва ІСНУЮЧОЇ віртуальної мережі"
+  type        = string
+}
+
+variable "subnet_name" {
+  description = "Назва ІСНУЮЧОЇ підмережі"
+  type        = string
+}
+
+variable "network_interface_name" {
+  description = "Назва ІСНУЮЧОГО мережевого інтерфейсу"
+  type        = string
+}
+
+variable "virtual_machine_name" {
+  description = "Назва ІСНУЮЧОЇ віртуальної машини"
+  type        = string
+}
```

Consider adding `validation` blocks for important variables to fail early with a clear message if required variables are empty (for example ensure `resource_group_name` and `virtual_machine_name` are not empty). This improves reproducibility for graders/operators and avoids unclear prompts during plan/apply.